File: app/src/main/java/in/woloo/www/dailylogscreen/models/DailyLogSubTitle.java
package in.woloo.www.dailylogscreen.models;

import java.io.Serializable;

public class DailyLogSubTitle implements Serializable {
    public DailyLogSubTitle() {
    }

    public DailyLogSubTitle(String subTitleName, int imageUrl, boolean isChecked) {
        this.subTitleName = subTitleName;
        this.imageUrl = imageUrl;
        this.isChecked = isChecked;
    }

    public int id;
    public String subTitleName;
    public int imageUrl;
    public boolean isChecked = false;

    public boolean isChecked() {
        return isChecked;
    }

    public void setChecked(boolean checked) {
        isChecked = checked;
    }
}

File: app/src/main/java/in/woloo/www/more/dailylogscreen/models/DailyLogSubTitle.kt
package `in`.woloo.www.more.dailylogscreen.models

import java.io.Serializable

class DailyLogSubTitle : Serializable {
    constructor()

    constructor(subTitleName: String?, imageUrl: Int, isChecked: Boolean) {
        this.subTitleName = subTitleName
        this.imageUrl = imageUrl
        this.isChecked = isChecked
    }

    var id: Int = 0
    @JvmField
    var subTitleName: String? = null
    var imageUrl: Int = 0
    @JvmField
    var isChecked: Boolean = false
}
